server/internal/decoy/decoy.go

```diff
@@ -75,6 +75,7 @@ type decoy struct {
 	surbETAs   *avl.Tree
 	surbStore  map[uint64]*surbCtx
 	surbIDBase uint64
+	uniqueId   uint64
```

Comment:
```suggestion
```
We ended up just using the 8 random bytes from the SurbID instead, so we didn't need the sequential counter after all.